vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -1364,6 +1372,45 @@ void LedgerImpl::GetConfirmationsHistory(
       to_timestamp_seconds, callback);
 }
 
+void LedgerImpl::RefreshPublisherVerifiedStatus(
+    const std::string& publisher_key,
+    ledger::OnRefreshPublisherVerifiedStatusCallback callback) {
+  bat_publishers_->RefreshPublisherVerifiedStatus(publisher_key, callback);
+}
+
+void LedgerImpl::PanelLoadPublisherList(
+    uint64_t window_id,
+    const std::string& publisher_key) {
+  DownloadPublisherList(
+      std::bind(&LedgerImpl::OnPanelPublisherListDownloaded,
+        this, _1, _2, _3, window_id, publisher_key));
+}
+
+void LedgerImpl::OnPanelPublisherListDownloaded(
+    int response_status_code,
+    const std::string& response,
+    const std::map<std::string, std::string>& headers,
+    uint64_t window_id,
+    const std::string& publisher_key) {
+  if (response_status_code != 200) {
+
```

Comment:
we need to report this back to the panel that we were not successful

Reply:
This was meant to be removed. We pass `response_status_code` into `LoadPublishersListCallback` to retry the download. Is this sufficient?